application_sdk/clients/sql.py

```diff
@@ -236,6 +236,21 @@ def add_connection_params(
 
         return connection_string
 
+    def update_dialect_in_url(self, sqlalchemy_url: str) -> str:
```

Comment:
this method needs a better name - `get_supported_sqlalchemy_url`?

the method just returns the url which is compatible with the driver underneath. update implies it's updating one of the instance properties or variable.

Reply:
Done